t/t0013-config-file.t

```diff
@@ -177,6 +201,33 @@ test_expect_success 'start size=2 instance with ipc://' '
 	test_cmp ipc.exp ipc.out
 '
 
+test_expect_success 'start size=3 instance with ipc:// and custom topology' '
+	BINDDIR=$(mktemp -d) &&
+	test_when_finished "rm -rf $BINDIR" &&
+	mkdir conf8a &&
+	cat <<-EOT >conf8a/bootstrap.toml &&
+	[bootstrap]
+	curve_cert = "testcert"
+	[[bootstrap.hosts]]
+	host = "fake0"
+	bind = "ipc://${BINDDIR}/fake0"
+	connect = "ipc://${BINDDIR}/fake0"
+	[[bootstrap.hosts]]
+	host="fake1"
+	bind = "ipc://${BINDDIR}/fake1"
+	connect = "ipc://${BINDDIR}/fake1"
+	[[bootstrap.hosts]]
+	host="fake2"
+	parent = "fake1"
```

Comment:
nit: why spaces around equal signs sometimes but not others? :-)

Reply:
Typo!